build-settings-logic/src/main/kotlin/atomicfu-gradle-build-scan.settings.gradle.kts

```diff
@@ -7,8 +7,8 @@ plugins {
 develocity {
     if (buildScanEnabled.get()) {
         val overriddenName = buildScanUsername.orNull
+        server = "https://ge.jetbrains.com/"
```

Comment:
nit: maybe it worth to left a comment, why this line here and not in `buildScan` for future generations